gobblin-modules/gobblin-compliance/src/main/java/gobblin/compliance/utils/DatasetUtils.java

```diff
@@ -44,20 +44,20 @@
     return Optional.<HivePartitionDataset>absent();
   }
 
-  public static String getProperty(HivePartitionDataset dataset, String property, int defaultValue) {
+  public static String getProperty(HivePartitionDataset dataset, String property, long defaultValue) {
```

Comment:
I added this method because it is used at various places in gobblin-compliance to read the value of a property from partition parameters. I made it string because that was the use case.